react/features/conference/functions.js

```diff
@@ -1,11 +1,30 @@
+import { getName } from '../app/functions.web';
```

Comment:
This is likely going to cause mobile to break because it probably also imports stuff from this file.

Reply:
Even if we don't (yet), can you please split it in functions.web / functions.native ?